megamek/src/megamek/common/AmmoType.java

```diff
@@ -179,6 +179,12 @@
     public static final BigInteger F_NUCLEAR = BigInteger.valueOf(1).shiftLeft(
             9); // Nuclear
     // missile
+    public static final BigInteger F_SANTA_ANNA = BigInteger.valueOf(1).shiftLeft(
+            14); // Santa Anna Missile
+    
+    public static final BigInteger F_PEACEMAKER = BigInteger.valueOf(1).shiftLeft(
+            14); // Peacemaker Missile
+
```

Comment:
Both of these shouldn't be 14.  We need to have distinct flags.

Reply:
Oops. I knew better, just missed that one. Fixed now.